### PR TITLE
Clarify low memory prompt #1182

### PR DESCRIPTION
**Pull request by rjhornsby**
Clarify whiptail prompt to include a specific error message re insufficient memory

Fix https://github.com/nextcloud/vm/issues/1182

### CODE REVIEW

File: lib.sh [hunk 1]
```diff
@@ -631,18 +631,20 @@ fi
 ram_check() {
 install_if_not bc
 mem_available="$(awk '/MemTotal/{print $2}' /proc/meminfo)"
-mem_available_gb="$(echo "scale=2; $mem_available/(1024*1024)" | bc)"
+mem_available_gb="$(printf '%0.2f\n' $(echo "scale=3; $mem_available/(1024*1024)" | bc))"
```
Comment (enoch85) [inline lib.sh:634]: Why this change if I may ask?

Comment (rjhornsby) [reply lib.sh:634]: It adds a leading zero, so instead of 

> .94GB installed, 2GB required

you get

> 0.94GB installed, 2GB required

which more closely aligns the formatting between the two numbers. When looking the rendered whiptail dialog, it was a bit more readable with the leading zero when quickly mentally comparing `0.94` and `2`

Screen Shot 2020-05-04 at 15 31 29

Comment (enoch85) [reply lib.sh:634]: OK, tested now and it looks good as far as I've tested.

**Comment by enoch85** [comment]
Please fix travis with something like this: 

`mem_available_gb="$(printf '%0.2f\n' "$(echo "scale=3; $mem_available/(1024*1024)" | bc)")"`

Not tested with travis, but gets the correct output.

**Comment by enoch85** [comment]
Thank you very much for your contribution, and welcome as a contributor to the Nextcloud VM! :)
